assets/src/stories-editor/components/post-selector/index.js

```diff
@@ -0,0 +1,339 @@
+/**
+ * PostSelector component, largely based on the logic of the Upstream URLInput.
+ */
+
+/**
+ * External dependencies
+ */
+import { map, throttle } from 'lodash';
+import classnames from 'classnames';
+import scrollIntoView from 'dom-scroll-into-view';
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf, _n } from '@wordpress/i18n';
+import { Component, createRef } from '@wordpress/element';
+import { UP, DOWN, ENTER, TAB } from '@wordpress/keycodes';
+import { Spinner, withSpokenMessages, Popover } from '@wordpress/components';
+import { withInstanceId, withSafeTimeout, compose } from '@wordpress/compose';
+import apiFetch from '@wordpress/api-fetch';
+import { addQueryArgs } from '@wordpress/url';
+import { decodeEntities } from '@wordpress/html-entities';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import './edit.css';
+
+const stopEventPropagation = ( event ) => event.stopPropagation();
+
+class PostSelector extends Component {
```

Comment:
Add a comment here.

Reply:
There is already one at the top of the file FWIW